generators/entity/templates/server/src/main/java/package/domain/_Entity.java

```diff
@@ -91,7 +93,9 @@
 @Document(collection = "<%= entityTableName %>")
 <%_ } if (databaseType === 'cassandra') { _%>
 @Table(name = "<%= entityInstance %>")
-<%_ } if (searchEngine === 'elasticsearch') { _%>
+<%_ } if (searchEngine === 'elasticsearch' && databaseType === 'mongodb') { _%>
+@org.springframework.data.elasticsearch.annotations.Document(indexName = "<%= entityInstance.toLowerCase() %>")
```

Comment:
While this is correct my idea was to simplify this with a variable. Dont worry ill do it while merging. Ill merge this later

Reply:
I think this way offers better readability of what's going in the template unless you want to put the logic here, but it's up to you. I don't have a strong opinion about it.